src/Behaviours/Cacheable.php

```diff
@@ -32,6 +32,30 @@ public function updateCacheRecord(array $config, $operation, $amount, $foreignKe
         return DB::update($sql);
     }
 
+    public function rebuildCacheRecord(array $config, Model $model, $command, $aggregateField = null)
+    {
+
+        $table = $this->getModelTable($config['model']);
+        $modelTable = $this->getModelTable($model);
+
+        // the following is required for camel-cased models, in case users are defining their attributes as camelCase
+        $field = snake_case($config['field']);
```

Comment:
Not sure how you suggest doing that.  A method that takes in a variable number of arguments by ref, and snake_cases them all?